scripts/crowbar-backup-cronjob

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# example backup script to be added in /etc/cron.daily/
+dir=/root/backup
+btarballname=crowbar-$(date +%F)
+p="--password=crowbar"
+mkdir -p $dir
+cd $dir
+crowbarctl backup create $btarballname $p && \
+bid=`crowbarctl backup  list --plain $p | grep ${btarballname} | cut -d" " -f1` && \
+crowbarctl backup download $bid $p
```

Comment:
because crowbarctl did not tell that it wrote it to `/var/lib/crowbar/backup/`. will update the script.